lib/command/generate.js

```diff
@@ -82,22 +85,32 @@ module.exports.pageObject = function (genPath, opts) {
   output.print(`Creating a new ${kind} object`);
   output.print('--------------------------');
 
-  return inquirer.prompt([{
-    type: 'input',
-    name: 'name',
-    message: `Name of a ${kind} object`,
-    validate: (val) => !!val,
-  }, {
-    type: 'input',
-    name: 'filename',
-    message: 'Where should it be stored',
-    default: answers => `./${kind}s/${answers.name}.js`,
-  }]).then((result) => {
+  return inquirer.prompt([
+    {
+      name: 'typescript',
+      type: 'confirm',
+      default: false,
+      message: `Do you plan to write ${kind} object in TypeScript?`,
```

Comment:
we shell not ask this but guess from a config extension
if it is `codecept.conf.ts` then it should be `ts` extension